packages/moonstone/Spinner/Spinner.module.less

```diff
@@ -161,35 +110,45 @@
 		padding: ((@moon-button-height - @moon-spinner-size) / 2);
 	}
 
-	&.running .ball {
-		-webkit-animation-play-state: running;
-		animation-play-state: running;
+	&.running .decorator {
+		// Apply the following rules to the above selector and all of these children
+		&,
+		.fan1::before,
+		.fan1::after,
+		.fan2::before,
+		.fan2::after,
+		.cap {
+			-webkit-animation-play-state: running;
+			animation-play-state: running;
+		}
 	}
 
 	// Skin colors
 	.applySkins({
 		color: @moon-spinner-text-color;
 
-		.ballDecorator {
+		.bg {
 			background-color: @moon-spinner-bg-color;
 		}
 
-		&.transparent {
-			.ballDecorator {
-				background-color: transparent;
-			}
-		}
-
-		.ball1 {
-			color: @moon-spinner-ball1-color;
-		}
+		.decorator {
+			background: conic-gradient(@moon-spinner-empty-color 75%, @moon-spinner-color 87.5%);
```

Comment:
Just noting for doc purposes that firefox doesn't yet support this :(

https://bugzilla.mozilla.org/show_bug.cgi?id=1175958

Reply:
Boo. Though I'm curious how many things we do elsewhere that FireFox doesn't support.